RT_UPDATE/a0_identification.R

```diff
@@ -0,0 +1,47 @@
+# Andreev–Kingkade a0
+a0_AK <- function(m0, sex = "f") {
+  
+  lt_rule_ak_m0_a0(M0 = m0, Sex = sex)
+  
+}
+
+
+# Solve for a0 given L0 
+solve_a0_from_L0 <- function(L0_target, 
```

Comment:
change name to standard DemoTools naming convention. something like lt_rule_L0_a0()